### PR TITLE
exposes the `.getMaker()` method publicly and adds missing tests

### PR DESCRIPTION
**Pull request by johnspackman**
Apologies for the late entry on this one, I intended for this to be part of the last PR.

The examples call the `._getMaker()` method of the command in order to integrate fully with the compiler API, but this is not ideal given that it is a protected method.  The maker is an essential entry point, so this change exposes it publicly.

Once this is merged, I'll update API viewer and Demo browser to match

### CODE REVIEW

File: test/testapp/compile.js [hunk 1]
```diff
@@ -19,11 +21,64 @@ qx.Class.define("qxl.compilertests.testapp.LibraryApi", {
...
+    __startedSassWatch: false,
+    
     async load() {
       let command = this.getCompilerApi().getCommand();
+      command.addListener("making", () => this._onMaking());
       command.addListener("checkEnvironment", e => this._appCompiling(e.getData().application, e.getData().environment));
     },
     
+    _onMaking() {
+      debugger;
```
Comment (cboulanger) [inline test/testapp/compile.js]: Is this here on purpose? I haven't used this in a while, but somehow remember it would stop execution...

Comment (johnspackman) [reply test/testapp/compile.js]: it only stops (pauses) under the debugger, but it shouldnt be there - removing it now

**Comment by hkollmann** [comment]
@johnspackman: Could you resolve the conflict and have a look at my Question please?

File: package.json [hunk 2]
```diff
@@ -75,6 +75,7 @@
     "puppeteer-to-istanbul": "^1.2.2",
     "replace-in-file": "^4.0.0",
     "rimraf": "^2.6.2",
+    "runscript": "^1.3.1",
```
[the remark this answers is not in the document]
Comment (johnspackman) [reply package.json]: done

**Pull request closed without merging**